release.sh

```diff
@@ -0,0 +1,189 @@
+#!/bin/bash
+
+DEFAULT_REPO="cimnine/netbox-docker"
+# DEFAULT_REPO=netbox-community/netbox-docker
```

Comment:
I think this should be the active default. Most of the time a new release will be made from develop of the `netbox-community/netbox-docker` repository.